.github/scripts/test-offline-diacritization.sh

```
@@ -0,0 +1,60 @@
+set -ex
+
+log() {
+  # This function is from espnet
+  local fname=${BASH_SOURCE[1]##*/}
+  echo -e "$(date '+%Y-%m-%d %H:%M:%S') (${fname}:${BASH_LINENO[0]}:${FUNCNAME[1]}) $*"
+}
+
+echo "EXE is $EXE"
+echo "PATH: $PATH"
+
+which $EXE
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)